Increased the lifetime of the script in the cache.

chuikoaleksandr opened the pull request:
Now scripts that have not been executed for more than an hour are cleared from the cache. Because of this, freezes occur during the day during the initialization of large scripts.

chuikoaleksandr committed:
Increased the lifetime of the script in the cache.
Now scripts that have not been executed for more than an hour are cleared from the cache. Because of this, freezes occur during the day during the initialization of large scripts.

mashintsev commented on src/main/java/ru/mail/jira/plugins/groovy/impl/ScriptServiceImpl.java:
```diff
@@ -56,7 +56,7 @@
     private final Cache<String, CompiledScript> scriptCache = Caffeine
         .newBuilder()
         .maximumSize(1000)
```
нужно убрать эту настройку

chuikoaleksandr committed:
Increased the lifetime of the script in the cache.
Now scripts that have not been executed for more than an hour are cleared from the cache. Because of this, freezes occur during the day during the initialization of large scripts.
